grammars/javascript.cson

```diff
@@ -724,6 +724,41 @@
     ]
   }
   {
+    # Math
+    'begin': '\\bMath\\b'
```

Comment:
Yep, and these bugs are sorta-kinda documented at #344 :wink:.

I guess we'll just have to fix these issues in another PR.